website/docs/src/content/index.mdx

```diff
@@ -42,6 +42,76 @@ LLM workflows are fundamentally about function composition and data flow - exact
 
 While GenSX uses JSX, it does not share the same execution model as React, and has zero dependencies.
 
+## GenSX Cloud
+
+[GenSX Cloud](/docs/cloud) provides everything you need to ship production grade agents and workflow including a serverless runtime designed for long-running workloads, cloud storage to build stateful workflows and agents, tracing and observability, and features for team collaboration.
```

Comment:
🔥 Love this framimg. Super compelling. 

I would take out the "features for team collaboration". This is essentially your thesis so the sections below should align with this sentence. 
```suggestion
[GenSX Cloud](/docs/cloud) provides everything you need to ship production grade agents and workflow including a serverless runtime designed for long-running workloads, cloud storage to build stateful workflows and agents, and tracing and observability.
```